sdk-tests/pom.xml

```diff
@@ -65,6 +65,12 @@
       <version>2.2.2.RELEASE</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.javafaker</groupId>
```

Comment:
This seems to be a convenient package to generate data. We've finished a review of 3rd party dependencies. I do not see this one adding enough value to justify adding another 3rd party dependency.

Reply:
That was removed